operations/commit_queue.go

```diff
@@ -193,14 +193,19 @@ type mergeParams struct {
 }
 
 func (p *mergeParams) mergeBranch(ctx context.Context, conf *ClientSettings, client client.Communicator, ac *legacyClient) error {
+	projectRef, err := ValidateProjectID(conf, ac, p.projectID, p.skipConfirm)
```

Comment:
maybe we could get a projectRef from the AC and then make ValidateProjectID() work without needing the legacy client.

Reply:
I was thinking checking the project exists is part of validation.

Reply:
it could go either way, and it makes the code cleaner to not have to pass the legacy client.